shell/browser/browser_process_impl.h

```diff
@@ -100,7 +100,9 @@ class BrowserProcessImpl : public BrowserProcess {
   void StartAutoupdateTimer() override {}
 #endif
   void SetApplicationLocale(const std::string& locale) override;
+  void SetSystemLocale(const std::string& locale);
```

Comment:
nit: move the non-virtual methods to the top before the overrides to be consistent.